public/js/weather.js

```diff
@@ -0,0 +1,70 @@
+// Set API key, longitude, and latitude
+const api = "232f1a91a4893327ac3317c3786ac2be";
```

Comment:
Please remove the hardcoded API key from the file and instead add it to the environment variable.